content/2018/justinmk-exmode.md

```diff
@@ -267,9 +263,8 @@ fi
 ## Eggnog
 
 The mechanisms and ergonomics for delivering data to Vim at invocation-time are
-essentially unchanged from vi—owing, yes, to deference to
-[POSIX vi](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/vi.html)
-and our old friend backwards compatibility—but perhaps primarly to inertia.
+essentially unchanged from vi—owing, yes, to deference to POSIX and our old
```

Comment:
posix URL is already linked earlier in the article, so it's redundant to link it here.